### PR TITLE
 #65 - Create the login screen

### LINKED ISSUE #65: Create the login screen
## Description:

Implement the login screen according to the updated design.

Login

### PR DESCRIPTION
**Pull request by Koushith**
## Description

- [x] Implemented login and registration Screen
- [x] Replaced old logo with New Gradient Logo.
- [x] Localization of text
- [x] Responsiveness

## Screenshot
image
image
image
image

## Platform tested
- [x] Web
- [x] Tablet

### CODE REVIEW

File: src/screens/auth/login/login.screen.tsx [hunk 1]
```diff
@@ -1,15 +1,48 @@
...
+          />
+
+          <SocialLoginContainer>
+            <TextContainer>
+              <Text variant='contentHeader' center tx='common.or' />
+              <Text variant='contentHeader' center tx='common.alternativeOption' />
+            </TextContainer>
+
+            <SocialIconsContainer>
+              <Image name='LoginWithGitHub' height={5} width={7} />
```
Comment (koshikraj) [inline src/screens/auth/login/login.screen.tsx]: These should be should have the functionality of `button`.

Comment (Koushith) [reply src/screens/auth/login/login.screen.tsx]: yeah, Image supports `onClick ` event handler.

Comment (koshikraj) [reply src/screens/auth/login/login.screen.tsx]: Okay, then we need an optional prop to enable `cursor: pointer`.

Comment (Koushith) [reply src/screens/auth/login/login.screen.tsx]: done. pushed in the latest commit.

File: src/components/common/auth-header.component.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+      <StyledNav>
+        <List>
+          <Text variant='small'>Don't know what Safient does?</Text>
+        </List>
+        <List>
+          <Button
+            variant='ghost'
+            label={{ text: 'Know more', color: 'textLight' }}
+            onClick={() => window.location.href = "https://docs.safient.io"}
```
Comment (dsouzaedison) [inline src/components/common/auth-header.component.tsx]: Don't hard code values like this. Either use local of global constants.

File: src/components/common/auth-header.component.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+export const Header = () => {
+  return (
+    <NavContainer>
+      <LogoWrapper>
+        <Image src={safientLogo} width={12.1} />
+      </LogoWrapper>
+
+      <StyledNav>
+        <List>
+          <Text variant='small'>Don't know what Safient does?</Text>
```
Comment (dsouzaedison) [inline src/components/common/auth-header.component.tsx]: Use localisation

File: src/components/common/auth-header.component.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        </List>
+        <List>
+          <Button
+            variant='ghost'
+            label={{ text: 'Know more', color: 'textLight' }}
```
Comment (dsouzaedison) [inline src/components/common/auth-header.component.tsx]: Localise

File: src/navigation/navigation.tsx [hunk 5]
```diff
@@ -8,7 +9,8 @@ import { routes } from './routes';
 
 const Navigation = observer(() => {
   const { accountStore } = useStores();
-  const isLoggedIn: boolean = !!accountStore.getUser();
+  const isMock: boolean = true
```
Comment (dsouzaedison) [inline src/navigation/navigation.tsx]: Is this supposed to be committed?

**Review by dsouzaedison — DISMISSED** [review]

File: src/navigation/navigation.tsx (hunk 5 above)
Comment (koshikraj) [reply src/navigation/navigation.tsx]: I have added this to render the private pages as well until we integrate authentication. We can add a TODO.

File: src/components/common/auth-header.component.tsx (hunk 2 above)
Comment (Koushith) [reply src/components/common/auth-header.component.tsx]: fixed in the latest commit.

**Review by koshikraj — APPROVED** [review]

**Pull request merged**